### PR TITLE
Make CRUD SQL queries for Users Table

### LINKED ISSUE #12: Make CRUD SQL queries for Users table
## Instructions
- Write the SQL queries associated with CRUD (create, read, update, delete) routes for the `users` endpoint.

## Notes
- You can’t test the queries at the moment so just try your best and leave the queries in a temporary file under `server/queries/users.sql`.

## Acceptance Criteria
- `GET` - Returns all data from the Users table
- `GET/pending-accounts` - Returns all data from Users table who are currently pending approval
- `POST` - Adds a new row into `Users` table
- `PUT/:uid` - Updates an existing user as approved
- `DELETE/:uid` - Deletes an existing user given their id

## Resources
- [DB Model](https://www.figma.com/file/NDPdo1MR13qgsB4LubtzyZ/Database-Model?type=whiteboard&node-id=0-1&t=m1J0vcLI2mzIb5SN-0)
- [Postgres Select](https://www.postgresqltutorial.com/postgresql-tutorial/postgresql-select/)
- [Postgres Insert](https://www.postgresqltutorial.com/postgresql-tutorial/postgresql-insert/)
- [Postgres Update](https://www.postgresqltutorial.com/postgresql-tutorial/postgresql-update/)
- [Postgres Delete](https://www.postgresqltutorial.com/postgresql-tutorial/postgresql-delete/)

### PR DESCRIPTION
**Pull request by ctc-devops**
Authors:
 ### What does this PR contain?
 ### How did you test these changes?
 ### Attach images (if applicable)
 Closes #12

### CODE REVIEW

File: server/queries/users.sql [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+-- DELETE/:uid - Deletes an existing user given their id
+DELETE FROM
+    users
+WHERE
+    id = ?;
+
+-- GET - Returns all data from the Users table
+SELECT * FROM
+    Users;
```
Comment (michellelin1) [inline server/queries/users.sql]: good work! just a minor fix: the `users` table is lower case (sorry for the inconsistency in the directions). but other than that, the code looks good!

File: .vscode/settings.json
Comment (michellelin1) [inline .vscode/settings.json:1]: also please remove this file

**Review by michellelin1 — CHANGES_REQUESTED** [review]

File: server/queries/users.sql (hunk 1 above)
Comment (Philip-Jian) [reply server/queries/users.sql]: oh okay got it I was confused about that, thanks for clarifying.

**Comment by seancfong** [comment]
@michellelin1 @ThatMegamind Requesting a re-review with new changes, thanks!

**Review by michellelin1 — APPROVED** [review]
LGTM! good work :)

**Pull request merged**